### ISSUE TITLE
When manually updating the schedule, the 'Today' tag should be removed from rescheduled tasks

### ISSUE THREAD

**Issue by apkawel** [ksalzke/scheduling-omnifocus-plugin#2]
- Tasks that are marked with the 'Today' tag and that are later rescheduled seem to retain the 'Today' tag in addition to the new (re)scheduled tag.
- Manually updating the schedule does not seem to remove the 'Today' tag.
- I believe that the 'Today' tag is only removed after midnight (when the next run of the script changes the scheduled tags).
- It would be nice if this were done automatically throughout the day (I'm running the update script periodically through Keyboard Maestro) instead of having to either do it manually or wait until midnight passes.

**Comment by ksalzke** [comment]
I’ve now resolved the first point - that was a bug!

In relation to the other points:

As it stands, the script doesn’t remove the Today tag after it has been added. For my own use, tasks marked ‘Today’ stay there if they are not completed (i.e. to be completed tomorrow, or otherwise rescheduled). However I can see a use case where tasks marked ‘Today’ are have the tag/flag removed if they are not completed by the end of the day, and will leave this issue open as a potential option to add in the future.

Thanks!

**Comment by apkawel** [comment]
I could be misunderstanding, but I am still seeing the same behavior in v2.0.1: If I have a task tagged Today, and if I then use the plugin to reschedule it to another day, the Today tag does not get removed — even if I manually update the schedule.

I guess I don't understand why, in such a case, the Today tag should remain: the point is that the task is no longer scheduled for today, but rather for another day, and therefore I shouldn't see it scheduled for today anymore.

**Comment by apkawel** [comment]
I seem to have figured it out. For some reason, when I upgraded the plugin (which I tried doing several times) the new file did not actually copy over to the plugins folder — hence seeing the same behavior as before. Deleting the plugin and completely reinstalling the new version solved the problem. I just tested it, and behavior is as expected. Thank you!

One final note: to clarify, all four points in my original issue related to this particular bug that is now fixed. I agree with your point above that tasks tagged "Today" should stay tagged and included in the Today perspective for completion tomorrow etc. (In other words, I wasn't requesting a feature to remove the "Today" tag if the task isn't completed or rescheduled.) Just wanted to make sure I was explaining myself correctly! :-)
